src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -143,6 +143,27 @@ INSTANTIATE_TEST_SUITE_P(smoke_ConvolutionBackpropData2D_AutoPadding_OutputPaddi
                                 ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                         ConvolutionBackpropDataLayerTest::getTestCaseName);
 
+INSTANTIATE_TEST_CASE_P(smoke_ConvolutionBackpropData2D_RoundingOfPadding, ConvolutionBackpropDataLayerTest,
+                        ::testing::Combine(
+                                ::testing::Combine(
+                                        ::testing::ValuesIn(std::vector<std::vector<size_t>>({ {31, 1} })),
+                                        ::testing::ValuesIn(std::vector<std::vector<size_t>>({ {2, 1} })),
```

Comment:
```suggestion
                                ::testing::Combine(
                                        ::testing::Values({31, 1}),
                                        ::testing::Values({2, 1}),
```
The same is below. ValuesIn is usually used when we need to iterate through some existing container object.

Reply:
fix it

Reply:
@maxnick help to review